qutebrowser/browser/webkit/downloads.py

```diff
@@ -731,13 +738,30 @@ def __init__(self, win_id, parent=None):
         self._update_timer = usertypes.Timer(self, 'download-update')
         self._update_timer.timeout.connect(self.update_gui)
         self._update_timer.setInterval(REFRESH_INTERVAL)
+        self._tmpdir_obj = None
+
+    def cleanup(self):
+        """Clean up any temporary files from this manager."""
+        if self._tmpdir_obj is not None:
+            self._tmpdir_obj.cleanup()
+
+    @property
```

Comment:
Please make this a normal function - if there's any exception in it, with PyQt (due to it overriding `__getattribute__`) you'll get an `AttributeError: ... object has no attribute 'tmpdir'` instead of the real exception.